CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 # These owners will be the default owners for everything in
 # the repo.
-*       @BartoszCki @mkulaczkowski @kossak @dandruszak
\ No newline at end of file
+*       @Paperspace/gradient-deployments
```

Comment:
Just future-proofing a little since other product teams will probs be added as global owners as well at some point, Codeowners _should_ follow the same syntax as `.gitignore` files, so I think
```
**/*deployment*.py  @Paperspace/gradient-deployments
**/*model*.py       @Paperspace/gradient-deployments
```
should be valid and give our team more narrowly-scoped ownership to files we specifically care about with the benefit of directory globbing so we don't need to care where in the repo those files exactly live

Reply:
Interesting - didn't know github enforced owners like that!